manifests/oned/sunstone/config.pp

```diff
@@ -27,6 +27,8 @@
   $vnc_proxy_cert        = $one::vnc_proxy_cert,
   $vnc_proxy_key         = $one::vnc_proxy_key,
   $vnc_proxy_ipv6        = $one::vnc_proxy_ipv6,
+  $logo_png              = $one::sunstone_logo_png,
```

Comment:
please keep variable names identical.
changing variable names across modules makes it more dificult to read and understand the code.